CMakeLists.txt

```diff
@@ -7,10 +7,7 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
change:
```
set(CMAKE_CXX_STANDARD 17)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```
for:
`target_compile_features(bezier PUBLIC cxx_std_17)`

Reply:
and this line could be further down the file, where everything else is set per-target